src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,40 @@
+.footer {
+  position: relative;
+
+  background: url(../images/footer-bg.jpg) center/cover no-repeat;
```

Comment:
Everything okey, just wanna say that separate rules like `background-image`, `background-repeat`, `background-size` etc. looks more readable but for me its also okay))